python_modules/dagster-graphql/dagster_graphql/schema/partition_sets.py

```diff
@@ -121,6 +122,32 @@ class Meta:
         name = "PartitionStatusesOrError"
 
 
+class GraphenePartitionsStatus(graphene.Enum):
```

Comment:
Is this built from `BackfillPartitionsStatus`? If so, should it be `GrapheneBackfillPartitionsStatus`?